hasjob/templates/detail.html

```diff
@@ -315,6 +304,13 @@
         </p>
       </div>
     {%- endif %}
+    <div class="section hidden" id="viewstats">
+      <h2>Page views</h2>
+      <div><span id="post-viewed"></span> › <span id="post-opened"></span> › <span id="post-applied"></span> &middot; Per <span id="post-stats-unit"></span>:</div>
```

Comment:
Since we have space, we should include labels for each segment, plus placeholder text while the numbers load. I'd use three `&middot;` characters (···) so it appears like a raised ellipsis:

`··· Viewed › ··· Opened › ··· Applied`

I'd also move this entirely into the headline, removing the "Page views" label there.